docs/cody/capabilities/auto-edit.mdx

```diff
@@ -53,31 +89,56 @@ The following example demonstrates how to add Fireworks as an allowed LLM provid
-| ---------------------- | -------------------------------------- | -------------------------------------------------- |
-| **Text modification**  | Can only insert text, not delete       | Can both insert and delete text                    |
-| **Cursor position**    | Only proposes content after the cursor | Can modify code before and after the cursor        |
-| **Suggestion trigger** | Suggests on every character typed      | Proposes changes based on context and recent edits |
+- Click the three-dot menu on the top right of the Cody plugin window and select **Open Cody Settings Editor**
+- This will open the `cody_settings.json` file in your editor
+- Here, add `"cody.suggestions.mode": "auto-edit (Beta)"` setting to enable the auto-edit feature
 
-## Use cases
+![JetBrains-Cody-Settings-Editor](https://storage.googleapis.com/sourcegraph-assets/Docs/jb-cody-settings.png)
 
-The auto-edit feature can help you with various repetitive tasks in your code:
+### Auto-edit access for Enterprise customers
 
-- **Function signature refactoring**: Automatically suggests adding error returns, new parameters, or changing return types across function definitions. For example, when you modify a function to return both a value and an error, auto-edit helps update the function signature.
-- **Call site updates**: When you change a function's signature, auto-edit detects all locations where the function is called and suggests necessary modifications to match the new signature. This includes updating parameter orders, adding error handling, and adjusting return value usage.
-- **Test file maintenance**: Helps with repetitive updates in test files, such as modifying test assertions, updating mock objects, or changing test data structures. Auto-edit recognizes patterns from your recent changes and suggests similar modifications across related tests.
-- **Parameter refactoring**: Assists in adding, removing, or reorganizing function parameters. When you unpack a function to handle more cases, auto-edit helps restructure the parameter list and suggests corresponding changes at call sites.
-- **Type system modifications**: Auto-edit identifies and suggests consistent changes across your codebase when updating type definitions or interfaces. This includes updating variable declarations, function parameters, and return types to maintain type consistency.
+Auto-edit is available for Enterprise customers with [Sourcegraph Cody Gateway](/cody/core-concepts/cody-gateway#sourcegraph-cody-gateway) access. Enabling the feature requires two steps:
+
+1. Site administrators must:
+   - Ensure the feature flag `cody-autoedit-experiment-enabled-flag` is enabled (enabled by default)
+   - Add `fireworks::*` as an [allowed provider](https://sourcegraph.com/docs/cody/enterprise/model-configuration#model-filters) (see below)
+2. Once enabled, auto-edit will become the default suggestion mode for all users
```

Comment:
auto-edit won't be default in jetbrains. once enabled, users can opt in via the extension setting